client/src/Components/ProfileDropdown/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { AiTwotoneEdit } from 'react-icons/ai';
+import { IoLogOutOutline } from 'react-icons/io5';
+
+import './profiledropdn.scss';
+
+const ProfileDropdown = ({ handleEdit, handleExit }) => {
+  return (
+    <>
+      <section className="profDropDn">
```

Comment:
Not very important but maybe className="dropdown" is enough if you wish to make it more descriptive the className for scss are written in this format: prof-drop-dn not profDropDn